### ISSUE TITLE
[SF_ACC][0.28.1][Table][a11y] Cannot select/deselect a row by pressing space key when using JAWS 2022

### ISSUE THREAD

**Issue by i323808** [SAP/ui5-webcomponents#5806]
**Describe the bug**
Cannot select/deselect a row by pressing space key **when using JAWS 2022**.

**Isolated Example**
https://sap.github.io/ui5-webcomponents-react/?path=/docs/data-display-table--default-story

**To Reproduce**
1. Go to https://sap.github.io/ui5-webcomponents-react/?path=/docs/data-display-table--default-story
2. Switch mode to `MultiSelect`
3. **Open JAWS 2022**
4. Press space key to select or deselect row
5. Cannot select or deselect row by pressing space key

**Expected behavior**
User can select or deselect a row by pressing space key when using JAWS.

**UI5 Web Components for React Information**
`@ui5/webcomponents` version: 1.7.1
`@ui5/webcomponents-react` version: 0.28.1
Operating System: Windows 10
Browser: Chrome Version 104.0.5112.102
**Screen Reader: JAWS 2022**

**Comment by i323808** [comment]
Hi @Lukas742 ,

It's weird that the corresponding UI5 web component named Table is working fine when using JAWS 2022(https://sap.github.io/ui5-webcomponents/playground/components/Table).

BR,
Jane

**Comment by Lukas742** [comment]
Hi @i323808 

we don't change anything in the implementation of the web component, so it maybe has something to do with the storybook. Did you try rendering the table without iframe? --> https://sap.github.io/ui5-webcomponents-react/iframe.html?args=&id=data-display-table--default-story&viewMode=story

**Comment by i323808** [comment]
Hi @Lukas742 ,

Unfortunately, it also does not work without iframe. Actually QAS team found this issue with our application first. Then we found it is reproducible with UI5-wc-react demo.
By the way, the space key is working correctly **when close JAWS 2022**.

BR,
Jane

**Comment by Lukas742** [comment]
I created two codeSandbox examples, one with the React wrapper and one without. Could you please check these out and see if the behavior differs there as well?

- UI5WCR: [codeSandbox](https://codesandbox.io/s/great-hill-3iqv3k?file=/src/App.js) ([w/o iframe](https://3iqv3k.csb.app/))
- UI5WC: [codeSandbox](https://codesandbox.io/s/ui5-webcomponents-forked-fim3lg?file=/index.html) ([w/o iframe](https://fim3lg.csb.app/))

**Comment by i323808** [comment]
Both two examples do not work correctly when using JAWS 2022. So it is an issue of UI5 web component.

**Comment by tsanislavgatev** [comment]
Hello colleagues @SAP/ui5-webcomponents-topic-rl ,

Can you please take a look at this issue?

Best Regards,
Tsanislav

**Comment by niyap** [comment]
Hello Colleagues,

I tested the scenario and it works as expected.
Here is a sample of ui5-table: https://sap.github.io/ui5-webcomponents/playground/components/Table/
You are not able to select when JAWS is used in reading mode, but it is not expected to use space key for selection in reading mode. Please ensure that you are in application mode (virtual cursor off). In that case, it works as expected and the row could be selected via space key.

Kind Regards,
Niya
